cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
+	address varchar(64) NOT NULL,
+	latitude double(18,14) NOT NULL,
+	longitude double(18,14) NOT NULL,
+	rating float NOT NULL,
+	Monday char(11) NOT NULL,
+	Tuesday char(11) NOT NULL,
+	Wednesday char(11) NOT NULL,
+	Thursday char(11) NOT NULL,
+	Friday char(11) NOT NULL,
+	Saturday char(11) NOT NULL,
+	Sunday char(11) NOT NULL,
+	photourl text(512) NOT NULL,
+	take_out enum('YES', 'NO') NOT NULL,
+	parking enum('YES', 'NO') NOT NULL,
```

Comment:
We can do something similar here by using 0 for no parking and 1 otherwise. In this case maybe we can rename this to 'parking_available' or 'has_parking'?

Reply:
Same below

Reply:
I think parking_available and take_out_available will be good! Thanks a lot!